mobile/src/main/res/values-tr/strings.xml

```diff
@@ -57,31 +51,18 @@
-  <string name="error_invalid_url">Lütfen \'protocol://host:port/\' formunda geçerli bir URL giriniz</string>
   <string name="error_connection_failed">Ana cihaza bağlantı başarısız oldu</string>
   <string name="error_unable_to_resolve_hostname">Ana cihaz adı çözülemiyor</string>
-  <string name="error_connection_sslhandshake_failed">SSL Handshake başarısız oldu - Belki geçerli bir müşteri sertifikasını ihtiyacınız vardır</string>
   <string name="error_certificate_not_trusted">SSL sertifikasına güvenilmiyor. Lütfen kök sertifikayı içe aktarın ya da SSL handshake esnasında sertifika doğrulaması etkisiz hale getirin</string>
   <string name="error_certificate_expired">SSL sertifikasının süresi doldu. Lütfen yeni bir sertifika edinin ya da SSL handshake esnasında sertifika doğrulamasını etkisiz hale getirin</string>
-  <string name="error_certificate_not_valid_yet">SSL sertifikası henüz geçerli değil. Lütfen geçerli bir sertifika edinin ya da SSL handshake esnasında sertifika doğrulamasını etkisiz hale getirin. Ayrıca telefonunuzdaki saat ayarının doğruluğunu kontrol ediniz</string>
   <string name="error_certificate_revoked">SSL sertifikası iptal edildi. Lütfen yeni bir sertifika edinin ya da SSL handshake esnasında sertifika doğrulamasını etkisiz hale getirin</string>
-  <string name="error_http_code_401">Kullanıcı kimlik doğrulaması başarısız oldu. Lütfen kullanıcı adını, şifreyi ve SSL müşteri sertifikasını kontrol ediniz (HTTP yanıt kodu 401)</string>
-  <string name="error_http_code_403">Kullanıcı kimlik doğrulaması başarısız oldu. Lütfen kullanıcı adınızı, şifrenizi ve SSL müşteri sertifikanızı gözden geçirin (HTTP yanıt kodu 403)</string>
-  <string name="error_http_code_407">Kullanıcı kimlik doğrulaması başarısız oldu. Lütfen kullanıcı adınızı, şifrenizi ve SSL müşteri sertifikanızı gözden geçirin (HTTP yanıt kodu 403)</string>
-  <string name="error_http_code_414">openHAB URL\'si çok uzun (HTTP yanıt kodu 414)</string>
-  <string name="error_http_code_426">HTTPS gereklidir. Lütfen openHAB URL\'sini değiştirin (HTTP yanıt kodu 426)</string>
   <!-- Nginx specific codes, but since it is the recommended reverse proxy, we should check for them: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#nginx -->
-  <string name="error_http_code_495">Geçersiz SSL müşteri sertifikası (HTTP yanıt kodu 495)</string>
   <string name="error_http_code_496">SSL müşteri sertifikası gereklidir (HTTP yanıt kodu 496)</string>
-  <string name="error_http_code_502">Ters proxy\'iniz openHAB\'tan geçersiz bir yanıt aldı (HTTP yanıt kodu 502)</string>
-  <string name="error_http_code_503">openHAB geçici olarak kullanım dışı (HTTP yanıt kodu 503)</string>
-  <string name="error_http_code_504">Gateway zaman aşımı. Görünüşe göre sizin ters proxy\'iniz çalışıyor fakat openHAB\'dan bir cevap alamıyor (HTTP yanıt kodu 504)</string>
-  <string name="error_http_code_507">openHAB yeteri kadar boşluğa sahip değil (HTTP yanıt kodu 507)</string>
```

Comment:
@mueller-ma these deletions are the only concerning thing I saw. Besides that the PR seems ready to merge.